deploy/ocp-templates/glusters3template.json

```diff
@@ -0,0 +1,205 @@
+                            },
+                            {
+                                "name": "GLUSTER_USER",
+                                "value": "${GLUSTER_USER}"
+                            },
+                            {
+                                "name": "GLUSTER_PASSWORD",
+                                "value": "${GLUSTER_PASSWORD}"
+                            }
+                        ],
+                        "resources": {},
+                        "volumeMounts": [
+                            {
+                                "name": "gluster-vol1",
+                                "mountPath": "/mnt/gluster-object/${GLUSTER_VOLUMES}"
+                            },
+                            {
+                                "name": "gluster-vol2",
+                                "mountPath": "/mnt/gluster-object/gsmetadata"
+                            },
+                            {
+                                "name": "glusterfs-cgroup",
+                                "readOnly": true,
+                                "mountPath": "/sys/fs/cgroup"
+                            }
+                        ],
+                        "terminationMessagePath": "/dev/termination-log",
+                        "imagePullPolicy": "IfNotPresent",
+                        "securityContext": {
+                            "privileged": true
+                        }
+                    }
+                ],
+                "restartPolicy": "Always",
+                "terminationGracePeriodSeconds": 30,
+                "dnsPolicy": "ClusterFirst",
+                "serviceAccountName": "default",
+                "serviceAccount": "default",
+                "securityContext": {}
+            },
+            "status": {}
+        },
+        {
+            "kind": "Service",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "glusters3service",
+                "creationTimestamp": null,
+                "labels": {
+                    "name": "glusters3"
+                }
+            },
+            "spec": {
+                "ports": [
+                    {
+                        "protocol": "TCP",
+                        "port": 8080,
+                        "targetPort": 8080
+                    }
+                ],
+                "selector": {
+                    "name": "glusters3"
+                },
+                "type": "ClusterIP",
+                "sessionAffinity": "None"
+            },
+            "status": {
+                "loadBalancer": {}
+            }
+        },
+        {
+            "kind": "PersistentVolumeClaim",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "glusterfs-claim",
+                "creationTimestamp": null,
+                "annotations": {
+                    "volume.beta.kubernetes.io/storage-class": "s3storageclass"
```

Comment:
Name of storage class should be a parameter, with default value of s3storageclass perhaps?